typings/index.d.ts

```diff
@@ -1,5 +1,9 @@
 import * as React from 'react';
-import { AccessibilityState, AccessibilityStates, AccessibilityRole } from 'react-native';
+import {
```

Comment:
can we extract the Prettier run over `typings/` in a separate PR so that this one and #304 are shorter? :)

Reply:
Done